src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestEventCounter.Etw.cs

```diff
@@ -9,8 +9,7 @@ namespace BasicEventSourceTests
     public partial class TestEventCounter
     {
         // Specifies whether the process is elevated or not.
-        private static readonly Lazy<bool> s_isElevated = new Lazy<bool>(AdminHelpers.IsProcessElevated);
-        private static bool IsProcessElevated => s_isElevated.Value;
+        private static bool IsProcessElevated => Environment.IsPrivilegedProcess;
```

Comment:
I think there's a corresponding member on PlatformDetection, in which case case all of these IsProcessElevated helpers could be deleted, and the ConditionalFact/Theory attributes changed to use that (which is the primary purpose of PlatformDetection).  That consolidation could be done in a separate PR, though.

Reply:
Eh, I won't promise to do it later. I use VS Code (mostly for PowerShell dev) and it does not show dependencies in .Net sources. Maybe it's better in VS, but I haven't tried it.

Reply:
https://github.com/dotnet/runtime/blob/da1afc8c45afa7a980e5be450cfb1c65dc80f4d7/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs#L58
https://github.com/dotnet/runtime/blob/da1afc8c45afa7a980e5be450cfb1c65dc80f4d7/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Windows.cs#L256

Reply:
PR created: #78793